docs/conf.py

```diff
@@ -32,7 +32,7 @@
 
 # get absolute path to python project files
 python_project_path = pathlib.Path(os.path.abspath('../pypet2bids'))
-matlab_project_path = os.path.join(python_project_path.parent, 'matlab')
+matlab_project_path = os.path.join(python_project_path.parent)
```

Comment:
I think that this is necessary to then "pretend" that all the functions you want to document are in a "matlab" module.

See `matlab.rst`

Reply:
Ohh!! It's starting to come together now.